src/content/docs/en/guides/prefetch.mdx

```diff
@@ -154,6 +142,32 @@ export default defineConfig({
 });
 ```
 
+## Browser support
+
+Prefetching relies on [`<link rel="prefetch">`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/prefetch) if supported, or falls back to the [`fetch()` API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API). Below covers the common browsers which have subtle differences when prefetching:
+
+### Chrome
+
+1. It supports `<link rel="prefetch">` and will generally always work as expected.
+
+### Firefox
+
+1. It supports `<link rel="prefetch">` but errors with `NS_BINDING_ABORTED` if an explicit cache header like [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) or [`Expires`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires) doesn't exist.
+2. If it errors but the response have a proper [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag) header, it will be re-used on navigation.
+3. Otherwise, if it errors with no other cache headers, the prefetch will not work.
+
+### Safari
+
+1. It doesn't support `<link rel="prefetch">` and will fall back to the `fetch()` API.
+2. The `fetch()` API requires cache headers, like [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control), [`Expires`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires), and [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag), to be set in order to work. Otherwise, the prefetch will not work.
+3. Edge case: `ETag` headers do not work in private windows.
```

Comment:
```suggestion
**Support:** Safari does not support `<link rel="prefetch">` and will fall back to the `fetch()` API, which requires cache headers (e.g. [`Cache-Control`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control), [`Expires`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires), and [`ETag`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag)) to be set. Otherwise, the prefetch will not work.

**Edge case:** `ETag` headers do not work in private windows.
```